testdata/scripts/config_global.txt

```diff
@@ -0,0 +1,38 @@
+env HOME=$WORK/home
+
+gunk generate ./gunk
+
+# Check that that files were generated in the correct folder
+exists gunk/v1/all.pb.go gunk/v2/all_pb2.py
+
+-- go.mod --
+module testdata.tld/util
+
+-- gunk/v1/.empty --
+-- gunk/v2/.empty --
```

Comment:
why do we need these? if gunk is not creating the directories, I think it should be.

Reply:
Yup, `gunk generate` currently behaves similar to `protoc` which doesn't create the directories for you (which is why I think we went with this). If you think Gunk should do this, we should raise an issue.

Reply:
Ah, being consistent with protoc makes sense I guess. I find that weird of protoc, but at least we're being consistent.

Let's see if users complain about this, or if it's a problem in practice. I personally find it slightly annoying, but it's not a big deal.